### PR TITLE
New version: OpenWhisperSystems.Signal version 6.4.1

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Successful
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Signal | Signal 6.4.1    |
| Version     | 6.4.1     | 6.4.1 |
| Publisher   | Signal Messenger, LLC   | Signal Messenger, LLC      |
| ProductCode |           | 7d96caee-06e6-597c-9f2f-c7bb2e0948b4    |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95446)

### CODE REVIEW

File: manifests/o/OpenWhisperSystems/Signal/6.4.1/OpenWhisperSystems.Signal.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  InstallerUrl: https://updates.signal.org/desktop/signal-desktop-win-6.4.1.exe
+  InstallerSha256: C5F0C7B7E61F18F59D2E268827420B593ABCD414A44F238C2D595429C5D86A77
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://updates.signal.org/desktop/signal-desktop-win-6.4.1.exe
+  InstallerSha256: C5F0C7B7E61F18F59D2E268827420B593ABCD414A44F238C2D595429C5D86A77
+  InstallerSwitches:
+    Custom: /ALLUSERS
```
Comment (mdanish-kh) [inline manifests/o/OpenWhisperSystems/Signal/6.4.1/OpenWhisperSystems.Signal.installer.yaml:26]:
```suggestion
```
Signal does not support machine wide installation.

File: manifests/o/OpenWhisperSystems/Signal/6.4.1/OpenWhisperSystems.Signal.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2022-12-15
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://updates.signal.org/desktop/signal-desktop-win-6.4.1.exe
+  InstallerSha256: C5F0C7B7E61F18F59D2E268827420B593ABCD414A44F238C2D595429C5D86A77
+  InstallerSwitches:
+    Custom: /CURRENTUSER
```
Comment (mdanish-kh) [inline manifests/o/OpenWhisperSystems/Signal/6.4.1/OpenWhisperSystems.Signal.installer.yaml:20]:
```suggestion
```
Installer installs in user scope by default

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/o/OpenWhisperSystems/Signal/6.4.1/OpenWhisperSystems.Signal.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+InstallModes:
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2022-12-15
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://updates.signal.org/desktop/signal-desktop-win-6.4.1.exe
+  InstallerSha256: C5F0C7B7E61F18F59D2E268827420B593ABCD414A44F238C2D595429C5D86A77
+  InstallerSwitches:
```
Comment (mdanish-kh) [inline manifests/o/OpenWhisperSystems/Signal/6.4.1/OpenWhisperSystems.Signal.installer.yaml:19]:
```suggestion
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**